### PR TITLE
updated the readme

### PR DESCRIPTION
**Pull request by robertschwartz603**
this was a short test on pull requests

### CODE REVIEW

**Comment by MaxFrank13** [comment]
Looks good!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
 # CarryOn
 Group 2's 1st Project: a travel information web application.
+
+##
+
+This project will use API's to populate information fields about countries for travel purposes. 
+
+yada yada yada
```
Comment (justinasari) [inline README.md:8]: yada yada yada

**Review by justinasari — APPROVED** [review]

**Review by dokodesara — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (robertschwartz603) [reply README.md:8]: Im an architect!

**Review by MaxFrank13 — APPROVED** [review]
We're good to go!

**Pull request merged**